### DISCUSSION TITLE
Fire Emblem: Three Houses

### DISCUSSION THREAD

**Discussion by Jgkingu** [skyline-emu/skyline#629] [Game Requests]
fire emblem three houses

**Comment by PixelyIon** [comment]
This is a game I'm interesting in seeing work but likely won't get to for a while.

**Reply by Jgkingu** [reply]
Yep Probably not for sometime, i watched on YouTube and apparently in "that one" emulator the game Crashes in the battle phase, But I trust Skyline and for agame like this I Will wait 3 years if needed.
